tests/models/self_supervised/test_resnets.py

```diff
@@ -16,6 +16,7 @@
 )
 
 
+@torch.no_grad()
```

Comment:
Since for all the resnet models we don't compute backward. We shouldn't keep track of gradients, :smile: